provision/infinispan/commons.sh

```diff
@@ -0,0 +1,20 @@
+function error_and_exit() {
+    echo "Error: ${@:2}"
+    exit "${1}"
+}
+
+# required!
+[ -z "${CLUSTER_1}" ] && error_and_exit 1 "CLUSTER_1 is required. CLUSTER_1 is the name of the first ROSA cluster."
+[ -z "${NS_1}" ] && error_and_exit 3 "NS_1 is required. NS_1 is the namespace to install Infinispan in the first ROSA cluster"
+
+KUBECONFIG_1=${KUBECONFIG_1:-"./kubecfg_1"}
+KUBECONFIG_2=${KUBECONFIG_2:-"./kubecfg_2"}
+
+
+function rosa_oc_login() {
+    local kubecfg="${1}"
+    local cluster="${2}"
+
+    # if file exists, assume oc login is done
+    [ -f "${kubecfg}" ] || KUBECONFIG="${kubecfg}" CLUSTER_NAME="${cluster}" ${WD}/../aws/rosa_oc_login.sh
```

Comment:
This causes problems if `./kubecfg_1` and/or `./kubecfg_2` exist from the previous day as we'll attempt to contact ROSA cluster endpoints that no longer exist. A potential solution is to use `/tmp` here, alongside a timestamp in the filename to avoid conflicts.

Reply:
@ryanemerson - in the OpenShift taskfile, we use `KC_HOSTNAME_SUFFIX´ to see if the cluster has changed. Maybe this opens some possibilities to auto-detect this.